.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: c
```

Comment:
I think, it should be c++?

Reply:
In this case I'm not sure it mattered, but I did update it just in case.